boards/Kconfig

```diff
@@ -23,6 +23,9 @@ source "$(BOARD_DIR)/Kconfig.board"
 
 endchoice
 
+# Parse shields references
+# Don't do it as a menuconfig, as shield selection is a CMake feature.
+rsource "shields/*/Kconfig.shield"
```

Comment:
FWIW, I prefer `source` over `rsource` where `rsource` isn't needed, because it makes the path easier to grep for.

The globbing kinda breaks grepping already though.